thorlcr/graph/thgraphmaster.cpp

```diff
@@ -649,25 +649,48 @@ void CMasterActivity::done()
 
 void CMasterActivity::updateFileReadCostStats()
 {
-    // Updates numDiskReads & readCost in the file attributes and returns the readCost
-    auto updateReadCosts = [](IDistributedFile *file, CThorStatsCollection &stats)
+    // Returns updates numDiskReads & readCost in the file attributes and returns the readCost
```

Comment:
was this comment accidentally reverted? (the one on deleted line 652 seems correct)

Reply:
Yes, fixed now.